### ISSUE TITLE
`TypeError: Cannot set properties of undefined (setting '_parentID')`

### ISSUE THREAD

**Issue by jerrygreen** [davidfig/pixi-viewport#486]
I'm getting this error when trying to addChild:

```ts
const textureBunny = await Assets.load('https://pixijs.com/assets/bunny.png');
const bunny = new Sprite(textureBunny);
viewport.addChild(bunny) // <-- here the issue happens
```

Also, typescript complains about type mismatch:

`Argument of type 'Sprite' is not assignable to parameter of type 'DisplayObject'`.

Versions:

```json
"pixi-viewport": "5.1.0",
"pixi.js": "8.1.0"
```

It well might be this directly related to this issue:

- https://github.com/davidfig/pixi-viewport/issues/485

**Comment by ViniciusFXavier** [comment]
@jerrygreen pixi-viewport for now works with pixi.js version 8.0.4, with 8.1.0 has some break changes.

Try change pixi.js to `8.0.4`

**Comment by jerrygreen** [comment]
No, this bug seems to be related to the issue I referenced above, it has nothing to do with 8.0.4 or 8.0.10

I just installed pixi-viewport from GitHub master branch, and it turned out to work just fine. So it’s indeed bug with version on npm.

Maybe there are some incompatibilities with 8.0.10, but that’s not one of those.
